.github/workflows/ci.yml

```diff
@@ -56,7 +56,15 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1.2.0
 
+      - name: Install compiler for armhf arch
+        if: matrix.target == 'armv7-unknown-linux-gnueabihf'
+        run: |
+          sudo apt-get update
+          sudo apt-get install gcc-arm-linux-gnueabihf
+
       - name: Build binary
+        env:
+          CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: arm-linux-gnueabihf-gcc
```

Comment:
We could replace this by adding a `.cargo/config.toml` file to the repository and specifying the linker there: https://doc.rust-lang.org/cargo/reference/config.html#targettriplelinker

I think that is preferable because we don't specify it in the GitHub-specific workflow then which allows us to cross-compile easier on local machines.